sphinx/ext/latex_svg2pdf_converter.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+"""
+    sphinx.ext.latex_svg2pdf_converter
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    SVG to PDF converter extension for LaTeX builder
```

Comment:
Why only for LaTeX builder?

Reply:
HTML, ePUB and so on handle SVG fine (if the viewer supports it, but that's another story). AFAIK LaTeX is the only one that needs mandatory conversion to PDF. However we could provide a configuration option to fine-tune conversion rules.

Reply:
`ImageConveter` will be called only if the builder does not support the image format.
For example, HTML builder supports `image/svg+xml` format. So this converter will not be called on that case.
https://github.com/sphinx-doc/sphinx/blob/69383f83bd55fe2394e5f43392ff231b5c643b40/sphinx/builders/html.py#L227-L228

On the other hand, LaTeX builder does not support it. But it supports `application/pdf`.
https://github.com/sphinx-doc/sphinx/blob/69383f83bd55fe2394e5f43392ff231b5c643b40/sphinx/builders/latex.py#L57

`ImageConverter` class and its subclass will be called based on `conversion_rules` mapping.
```
+    conversion_rules = [
+        ('image/svg+xml', 'application/pdf'),
+    ]
```
This mapping table consists of list of `(source format, destination format)`.

For this reason, you don't need to restrict this only for LaTeX builder.

Reply:
Thanks, you're completely right. For some reason I must have missed that…